src/redux/reducers/dataReducer.js

```diff
@@ -25,24 +26,34 @@ export default function(state = initialState, action) {
         postList: action.payload,
         isLoading: false,
       };
+    case NEW_POST:
+      return {
+        ...state,
+        postList: [action.payload, ...state.postList],
+      };
     case LIKE_POST:
     case UNLIKE_POST:
-      let index = state.postList.findIndex(
+      let likeIndex = state.postList.findIndex(
         post => post.postId === action.payload.postId
       );
```

Comment:
I like the change of index title here to make things clearer. 👍

Reply:
Interestingly enough, it wouldn't let me reclarify index for both functions, since it was being initialized above, but wouldn't add the newly created index into the variable. So, I had to make them their own variables. Weird quirk there, redux.

Reply:
That's a recent thing JS does where it doesn't seem to scope stuff correctly with certain types of function... I don't really get it either, but the indexes are better this way.